blockchain/validators/src/test_utils.rs

```diff
@@ -26,3 +26,131 @@ pub fn make_metadata(seed: u64) -> BlockMetadata {
+        ],
+        http_body: String::from("{\"nonce\":\"fe89bd7386cdc13787dbb50b27917030\",\"id\":\"166196848040908016823295613654479498765\",\"timestamp\":\"2021-06-24T18:57:44.075285\",\"version\":4,\"epidPseudonym\":\"gKH0dexEpYfuyaGgaKKWmH4VJ8r0L3af1W//p6ya+WaN9BAlSW1Gj3NOWvrQIEAyLCof3fwS9pkLnZrYk3CXQC7VlPZs6EsMzqr40q2S7ObH6Qxq5SI6JNc18NIfOZ8+43l5j4yck2OD4g4J2fX5f2bnlaiWaRTcvtF7xnSCs/k=\",\"advisoryURL\":\"https://security-center.intel.com\",\"advisoryIDs\":[\"INTEL-SA-00334\"],\"isvEnclaveQuoteStatus\":\"SW_HARDENING_NEEDED\",\"isvEnclaveQuoteBody\":\"AgABABMMAAAMAAsAAAAAAL3lg34HPOkq5u/y0l94xuMAAAAAAAAAAAAAAAAAAAAAEREDBf+ABgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABQAAAAAAAAAHAAAAAAAAAGUyKK/SsCpsKPHcOxCLHfpFfRcLMq6OwpePlBvRZVyDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAsGlYcSrZMvAS/pEXN977Zsq1vawTTjTE382IrKf2zDgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADwd71Ni3xBrMF22K0CWRUZPzdE9GwSDkFoiuhQcee9R/Qyoz4HOFMtNl1HY74q8e0fWOh7yxmlply0fZ2eREZj\"}"),
+    };
+    (report1, report2)
+}
+
+/// Get a control Avr History Config using historical AVR data so we both
+/// verify serialization and verification works properly
+pub fn get_avr_history_config() -> AvrHistoryConfig {
+    let responder_id_1 = ResponderId::from_str("node1.prod.mobilecoinww.com::8443").unwrap();
+    let (avr1, avr2) = get_ias_reports();
+    let rec1 = AvrHistoryRecord {
+        responder_id: responder_id_1.clone(),
+        first_block_index: 0,
+        last_block_index: Some(480),
+        avr: None,
+    };
+    let rec2 = AvrHistoryRecord {
+        responder_id: responder_id_1.clone(),
+        first_block_index: 481,
+        last_block_index: Some(10399),
+        avr: Some(avr1),
```

Comment:
It looks like there is a a bit of dependency between `get_ias_reports()`, `get_avr_history_config()`, `SAMPLE_AVR_HISTORY_TOML` and `SAMPLE_AVR_HISTORY_JSON`.

Suggest creating a containing struct to group this logic, might consider having functions like
```rust
SampleAvrHistory::as_json();
SampleAvrHistory::as_toml();
SampleAvrHistory::as_config();
```
(not sold on the "as_x" names)